account/account.go

```diff
@@ -533,6 +533,27 @@ func (account *Account) WaitForTransactionReceipt(ctx context.Context, transacti
 	}
 }
 
+// AddTransaction calls dynamically one of Add<type>Transaction methods
+//
+// Parameters:
+// - ctx: the context.Context object for the transaction.
+// - txn: this Broadcast Transaction to be called.
```

Comment:
grama: "the Broadcast Transaction to be sent."**

Reply:
Done